build.ps1

```diff
@@ -217,6 +217,12 @@ $startTime = [DateTime]::UtcNow
 Write-Host "Build started at " $startTime
 Write-Host
 
+if ((Test-Path -Path "submodules/FileSystem/src") -eq $False)
```

Comment:
I'll update this to only call --init when the folder is missing, and update the rest of the time.